src/states.js

```diff
@@ -47,13 +47,14 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-/* exported httpPort, timeForContentLoaded, timeCorrection, boundListeners, TEMP_DISABLE_MEMORIES, globalStates, globalCanvas, publicDataCache, pocketItemId, globalScaleAdjustment, pocketFrame, pocketNode, globalDOMCache, shadowObjects, globalProgram, rotateX, rotationXMatrix, editingAnimationsMatrix, pocketDropAnimation, pocketBegin, visibleObjectTapInterval, visibleObjectTapDelay, overlayDiv, CRAFTING_GRID_WIDTH, CRAFTING_GRID_HEIGHT, DEBUG_DATACRAFTING */
+/* exported httpPort, defaultPort, timeForContentLoaded, timeCorrection, boundListeners, TEMP_DISABLE_MEMORIES, globalStates, globalCanvas, publicDataCache, pocketItemId, globalScaleAdjustment, pocketFrame, pocketNode, globalDOMCache, shadowObjects, globalProgram, rotateX, rotationXMatrix, editingAnimationsMatrix, pocketDropAnimation, pocketBegin, visibleObjectTapInterval, visibleObjectTapDelay, overlayDiv, CRAFTING_GRID_WIDTH, CRAFTING_GRID_HEIGHT, DEBUG_DATACRAFTING */
 
 /**********************************************************************************************************************
  ******************************************** constant settings *******************************************************
  **********************************************************************************************************************/
 
 var httpPort = 8080;
+var defaultPort = '8080';
```

Comment:
Is there a benefit to making this a string? It appears that other places specify ports as numbers instead

Reply:
Notably this has the potential to break things like `myPort === otherPort` if those ever crop up

Reply:
> Is there a benefit to making this a string? It appears that other places specify ports as numbers instead

It does not make a difference, as its always used as part of a string and therefore the number is translated to string. Since it is always used as part of a string, I saved it as string.

Reply:
> Notably this has the potential to break things like `myPort === otherPort` if those ever crop up

fixed